File: frontend/src/app/dto/answer.ts
export class Answer {
  id: number;
  answer: string;
  correct: Boolean;
  bonus: Boolean;

  constructor(args: Answer = <Answer>{}) {
    this.id = args.id;
    this.answer = args.answer;
    this.correct = args.correct;
    this.bonus = args.bonus;
  }

}

File: src/main/java/com/corelogic/schemaconverter/entity/Answer.java
package com.corelogic.schemaconverter.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.SequenceGenerator;

@javax.persistence.Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Answer {

    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "AnswerSeq")
    @SequenceGenerator(name = "AnswerSeq", sequenceName = "answer_id_seq", allocationSize = 1)
    @Id
    private Long id;

    private String answer;

    private Boolean correct;

}

